hardware/display/README.md

```diff
@@ -106,5 +106,8 @@ Read our troubleshooting steps, tips, and tricks here: [Raspberry Pi Touch Displ
 
 * Outer dimensions: 192.96 × 110.76mm
 * Viewable area: 154.08 × 85.92mm
-* [Download mechanical drawing (PDF)](mechanical/7InchDisplayDrawing-14092015.pdf)
-* [Additional drawing including radius and thickness of glass](mechanical/radius.png)
+* [Download mechanical drawing (PDF)](http://datasheets.raspberrypi.org/display/7-inch-display-mechanical-drawing.pdf)
```

Comment:
Should we be using https: links?  Otherwise LGTM :+1:

Reply:
Yes we should be using https://